### PR TITLE
Docs: Lonestar6 A100 GPUs (TACC)

### PR DESCRIPTION
**Pull request by ax3l**
Add a documentation on how to run on Lonestar6 A100 GPUs (TACC).

- [x] basic installation as application
- [x] install extra dependencies: c-blosc, ADIOS2, BLAS++, LAPACK++
- [x] PICMI installation for Python interface
- [ ] slurm script example & single-node aliases

cc @ycaophysics #3531 @rupenether @GivenChen

### CODE REVIEW

File: Tools/machines/lonestar6-tacc/lonestar6_warpx.profile.example [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+#   usage: runNode <command>
+alias runNode="srun -N 1 --ntasks-per-node=3 -t 0:30:00 -q interactive -C gpu --gpu-bind=single:1 -c 32 -G 4 -A $proj"
+
+# optimize CUDA compilation for A100
+export AMREX_CUDA_ARCH=8.0
+
+# optimize CPU microarchitecture for AMD EPYC 3rd Gen (Milan/Zen3)
+# note: the cc/CC/ftn wrappers below add those
+#export CXXFLAGS="-march=znver3"
+#export CFLAGS="-march=znver3"
```
Comment (ax3l) [inline Tools/machines/lonestar6-tacc/lonestar6_warpx.profile.example]: This should fix the linker issue (thx @WeiqunZhang):
```suggestion
#export CFLAGS="-march=znver3"
# avoid linker truncation errors (>2GB relocatable device object files)
# https://docs.nvidia.com/cuda/cuda-compiler-driver-nvcc/#host-linker-script-use-lcs-gen-lcs-hls
export CUDAFLAGS="--host-linker-script=use-lcs"
```

**Comment by lucafedeli88** [comment]
Hi @ax3l  ! I've noticed that this PR has not received any update since February 2023. Do you expect to work on it in the future or should we close it?

**Comment by ax3l** [comment]
attn @rupenether @GivenChen

File: Tools/machines/lonestar6-tacc/lonestar6_warpx.profile.example (hunk 1 above)
Comment (ax3l) [reply Tools/machines/lonestar6-tacc/lonestar6_warpx.profile.example]: Problem solved when going to gcc 11

**Comment by ax3l** [comment]
I took a look together with @rupenether and we found modules that compile well for A100. Updates pushed.

**Comment by ax3l** [comment]
Hi @rupenether, can you help me update the `getNode` and `runNode` aliases and the `lonestar6_a100.sbatch` script to finalize this PR? :)

**Review by EZoni — APPROVED** [review]

**Pull request merged**